src/na/device/Device.cpp

```diff
@@ -0,0 +1,491 @@
+/*
+ * Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+ * Copyright (c) 2025 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+/** @file
+ * @brief The MQT QDMI device implementation for neutral atom devices.
+ */
+
+#include "na/device/Device.hpp"
+
+#include "mqt_na_qdmi/device.h"
+#include "na/device/DeviceMemberInitializers.hpp"
```

Comment:
Feel free to simply resolve this, but given that the header is placed in `na/device` already, I would have named it `MemberInitializers.hpp`.